package.json

```diff
@@ -4,9 +4,9 @@
   "main": "lib/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "babel src --out-dir lib",
+    "build": "rollup -c scripts/build/rollup.config.js",
```

Comment:
this config file seems to have been removed!

Reply:
Sorry it did get checked in. There is a `build` directory in the .gitignore. What are we using that dir for?

Reply:
nothing right now, it's just generally the name of an output folder. I'd stick the tooling in a `/tools` directory